src/pages/workspaces/workspace/Data.js

```diff
@@ -1002,6 +1047,26 @@ const WorkspaceData = _.flow(
               }
             }, [`${type} (${typeDetails.count})`])
           }, sortedEntityPairs),
+          div({ style: Style.navList.heading }, ['Snapshots']),
+          _.map(([snapshotName, { id: snapshotId, entityMetadata: snapshotTables }]) => {
+            const snapshotTablePairs = _.flow(_.toPairs, _.sortBy(_.first))(snapshotTables)
+            return h(SnapshotCollapse, {
+              buttonStyle: { color: colors.dark(), fontWeight: 600 },
+              style: { fontSize: 14, marginRight: '0.5rem', lineHeight: '50px', padding: '0 1.5rem', borderBottom: `1px solid ${colors.dark(0.2)}` },
+              title: snapshotName
+            }, [
+              p({ style: { fontSize: 14, lineHeight: '1.5' } },
```

Comment:
It might be nicer here to use a div with explicit padding rather than a p. P elements have semantic meaning (here is a paragraph of text) that will confuse screen readers if they're used purely visually.